### ISSUE TITLE
ocp-index.el does not work on Cygwin

### ISSUE THREAD

**Issue by nojb** [OCamlPro/ocp-index#77]
The following regular expression is used to match filename + location info in `ocp-index.el`:
```
"^\\([^:]*\\):\\([0-9]\+\\):\\([0-9]\+\\)$"
```
This does not work on Windows where paths can start with "C:/", etc.  I propose changing the regexp to
```
"^\\(.*\\):\\([0-9]\+\\):\\([0-9]\+\\)$"
```
i.e., replace `"[^:]"` by `"."`.  If there are no opinions to the contrary I will submit a pull request with this tiny change.

**Comment by AltGr** [comment]
Hm, that should work. `:` being a valid character in Linux filenames too, actually, it would be better anyway!
Thanks

**Comment by nojb** [comment]
Closed via ca9fa5b022249933f1b04bcbd0778efe604faf67.
